src/libraries/System.Private.CoreLib/src/System/DateTime.cs

```diff
@@ -198,6 +202,68 @@ public DateTime(int year, int month, int day, Calendar calendar)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DateTime"/> structure to the specified year, month, day, hour, minute, second,
```

Comment:
Thank you for bringing the docs here. :thumbsup:

Reply:
With pleasure :smile: